src/styles/reset.scss

```
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
```

Instead of setting margin and padding to zero in all tags, you need to set them to their original position - for all 
elements: 
```suggestion
  margin: 0;
  padding: 0;
  box-sizing: border-box;
```